metadata.json

```diff
@@ -1,24 +1,62 @@
 {
-  "name": "luxflux-openvpn",
-  "version": "4.0.1",
-  "author": "luxflux",
+  "name": "puppet-openvpn",
+  "version": "4.1.0",
+  "author": "voxpupuli",
   "summary": "OpenVPN server puppet module",
   "license": "Apache-2.0",
-  "source": "https://github.com/luxflux/puppet-openvpn",
-  "project_page": "https://github.com/luxflux/puppet-openvpn",
-  "issues_url": "https://github.com/luxflux/puppet-openvpn/issues",
+  "source": "https://github.com/voxpupuli/puppet-openvpn",
+  "project_page": "https://github.com/voxpupuli/puppet-openvpn",
+  "issues_url": "https://github.com/voxpupuli/puppet-openvpn/issues",
   "description": "Puppet module to manage OpenVPN servers",
-  "tags": [ "vpn", "openvpn" ],
   "operatingsystem_support": [
-    { "operatingsystem": "Ubuntu", "operatingsystemrelease": ["12.04", "12.10", "13.04", "13.10", "14.04", "15.04"] },
-    { "operatingsystem": "Debian", "operatingsystemrelease": ["7", "8"] },
-    { "operatingsystem": "RedHat", "operatingsystemrelease": ["5", "6", "7"] },
-    { "operatingsystem": "CentOS", "operatingsystemrelease": ["5", "6", "7"] },
-    { "operatingsystem": "Archlinux" },
-    { "operatingsystem": "FreeBSD" }
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "14.04",
+        "15.04",
+        "16.04"
+      ]
+    },
+    {
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "7",
+        "8"
+      ]
+    },
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "6",
```

Comment:
Same here, support for it is still in there: https://github.com/wyardley/puppet-openvpn/blob/d74483948d1b74e2c1fdd5ee05e0b0fde7957ab4/manifests/params.pp#L40-L44